### PR TITLE
Added metric env var to compose

### PR DESCRIPTION
**Pull request by 3alpha**
Allow users to disable web3signer metrics for performance.

### CODE REVIEW

**Review by pablomendezroyo — APPROVED** [review]

**Review by pablomendezroyo — COMMENTED** [review]
Does it have a high impact the flag `--metrics-enabled`?

File: web3signer/entrypoint.sh [hunk 1]
```diff
@@ -92,7 +92,6 @@ exec /opt/web3signer/bin/web3signer \
   --http-listen-host=0.0.0.0 \
   --http-host-allowlist="web3signer.web3signer-prater.dappnode,ui.web3signer-prater.dappnode,prysm.migration-prater.dappnode,$ETH2_CLIENT_DNS" \
   --http-cors-origins=* \
-  --metrics-enabled=true \
```
Comment (pablomendezroyo) [inline web3signer/entrypoint.sh:95]: Should not be used the env `WEB3SIGNER_METRICS_ENABLED` as the value in the flag `--metrics-enabled`?

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

File: web3signer/entrypoint.sh (hunk 1 above)
Comment (3alpha) [reply web3signer/entrypoint.sh:95]: W3S should be configurable using environment variables as well: https://docs.eth2signer.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#metrics-enabled

**Comment by 3alpha** [comment]
According to w3s devs it may have an impact and I don't see a reason not to have that exposed.

File: web3signer/entrypoint.sh (hunk 1 above)
Comment (pablomendezroyo) [reply web3signer/entrypoint.sh:95]: I see, but this kind of breaks with the way of working we have with `EXTRA_OPTS`. Also you did not include any comment so its hard to know how it works

**Comment by pablomendezroyo** [comment]
Not needed

**Pull request closed without merging**